experimental/PropertyDDS/examples/partial-checkout/tsconfig.json

```diff
@@ -1,9 +1,11 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": [
+		"../../../../common/build/build-common/tsconfig.base.json",
+		"../../../../common/build/build-common/tsconfig.esm-only.json",
+	],
```

Comment:
@DLehenbauer , despite the name, this package was changed from ESM to CJS here.
It can be restored by setting "type": "module" in package.json. If problems, then go back to using "module": "esnext" in tsconfig.

Reply:
Good catch.  Should be address in #19582.